pnpm-lock.yaml

```diff
@@ -123,10 +124,11 @@ importers:
       '@babel/preset-env': 7.16.11_@babel+core@7.17.8
       '@babel/preset-react': 7.16.7_@babel+core@7.17.8
       '@babel/preset-typescript': 7.16.7_@babel+core@7.17.8
-      '@senecacdot/eslint-config-telescope': 1.1.0_eslint@7.32.0
+      '@senecacdot/eslint-config-telescope': link:tools/eslint
```

Comment:
We want to avoid this.  Rebasing should fix?

Reply:
I just rebased, I probably missed this